_posts/2020-12-04-a-new-way-to-learn-and-discover-quarkus.adoc

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title: A new way to learn & discover Quarkus (even for the Quarkus veterans)
+date: 2020-12-04
+tags: extensions codestarts quickstart
+synopsis: All our tooling has been updated and can now generate Quarkus application with example code showing the true power of the selected extensions...
+author: adamevin
+---
+
+What’s nice about this new feature is that it’s useful for fresh Quarkus developers as well as experienced ones. This is because of the wide range (and growing) of Quarkus extensions, you simply can’t know it all.
```

Comment:
```suggestion
What’s nice about codestarts is that it’s useful for fresh Quarkus developers as well as experienced ones. This is because of the wide range (and growing) of Quarkus extensions, you simply can’t know it all.
```